docs/content/how-to/baremetal-none.md

```diff
@@ -0,0 +1,427 @@
+RHEL8 doesn't include go1.17 officially but it can be installed via `gvm` by following the next steps:
+
+~~~sh
+# Install prerequisites
+sudo dnf install -y curl git make bison gcc glibc-devel
+git clone https://github.com/openshift/hypershift.git
+pushd hypershift
+ 
+# Install gvm to install go 1.17
+bash < <(curl -s -S -L https://raw.githubusercontent.com/moovweb/gvm/master/binscripts/gvm-installer)
+source ${HOME}/.gvm/scripts/gvm
+gvm install go1.17
+gvm use go1.17
+
+# build the binary
+make hypershift
+popd
+~~~
+
+Then, the `hypershift` binary can be moved to a convenient place as:
+
+~~~sh
+sudo install -m 0755 -o root -g root hypershift/bin/hypershift /usr/local/bin/hypershift
+~~~
+
+Alternatively, it can be compiled using a container as:
+
+~~~sh
+# Install prerequisites
+sudo dnf install podman -y
+# Compile hypershift
+mkdir -p ./tmp/ && \
+podman run -it -v ${PWD}/tmp:/var/tmp/hypershift-bin/:Z --rm docker.io/golang:1.17 sh -c \
+  'git clone --depth 1 https://github.com/openshift/hypershift.git /var/tmp/hypershift/ && \
+  cd /var/tmp/hypershift && \
+  make hypershift && \
+  cp bin/hypershift /var/tmp/hypershift-bin/'
+sudo install -m 0755 -o root -g root ./tmp/hypershift /usr/local/bin/hypershift
+~~~
+
+> **WARNING**: At the time of writting this document, there were some issues already fixed in HyperShift but unfortunately those weren't included in the latest release of the container.
+
+## Prerequisites: Create a custom HyperShift image
+
+To create a custom HyperShift image, the following steps can be performed:
+
+~~~sh
+QUAY_ACCOUNT='testuser'
+podman login -u ${QUAY_ACCOUNT} -p testpassword quay.io
+sudo dnf install -y curl git make bison gcc glibc-devel
+git clone https://github.com/openshift/hypershift.git
+pushd hypershift
+ 
+# Install gvm to install go 1.17
+bash < <(curl -s -S -L https://raw.githubusercontent.com/moovweb/gvm/master/binscripts/gvm-installer)
+source ${HOME}/.gvm/scripts/gvm
+gvm install go1.17 -B
+gvm use go1.17
+
+# Build the binaries and the container
+make build
+make RUNTIME=podman IMG=quay.io/${QUAY_ACCOUNT}/hypershift:latest docker-build docker-push
+
+sudo install -m 0755 -o root -g root hypershift/bin/hypershift /usr/local/bin/hypershift
+popd
+~~~
+
+## Deploy HyperShift
+
+Once the binary is in place, the operator deployment is performed as:
+
+~~~sh
+hypershift install
+~~~
+
+Or if using a custom image:
+
+~~~sh
+hypershift install --hypershift-image quay.io/${QUAY_ACCOUNT}/hypershift:latest
+~~~
+
+Using `hypershift install --render > hypershift-install.yaml` will create a yaml file with all the assets required to deploy HyperShift, then they can be applied as:
+
+~~~sh
+oc apply -f ./hypershift-install.yaml
+~~~
+
+## Deploy a hosted cluster
+
+There are two main CRDs to describe a hosted cluster:
+* [`HostedCluster`](https://hypershift-docs.netlify.app/reference/api/#hypershift.openshift.io/v1alpha1.HostedCluster) defines the control plane hosted in the management OpenShift
```

Comment:
need to inotruce line breaks for * to render properly

Reply:
Fixed! Thanks for your inputs.